save-money/src/views/Altex.vue

```diff
@@ -18,23 +19,24 @@ export default Vue.extend({
   components: {
     'products-list': ProductsList,
   },
-  data: () => ({
-    products: [],
-  }),
+  data() {
+    return {
+      products: [],
+    };
+  },
   created() {
-    this.getData();
+    this.startDataFetching();
   },
   methods: {
-    // TODO METHODS
-    getData() {
-      // api call to the server which retireves data
+    async startDataFetching() {
```

Comment:
So the logic is this:

You take the array endpoint and you want to loop thru it, so you know exactly what is the endpoint that you are trying to reach. 

`forEach` will loop thru each endpoint and for each endpoint you want to call it, and before you move to the next endpoint you want to use the interval to wait x seconds, in order to not spam the endpoint. Once the interval passes, you will continue with the iteration and the second endpoint will be called

Reply:
The problem with your approach is that for each endpoint your are replacing the products object, but basically what you wanna do is to add each response to the same object / array. So instead of rewriting it, you can take existing data and add the new ones to it. 

So one endpoint gets you 50 data for example, second data gets you another 50 and you need to have 100 in total, so you can see all

Reply:
If we go with the altex example, if you want to get the discounts for 10 categories, you will want to display all 10

Reply:
Let me know if you have any questions, good luck